src/workerd/io/compatibility-date.capnp

```diff
@@ -504,4 +504,10 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $compatDisableFlag("legacy_module_registry")
       $experimental;
   # Enables of the new module registry implementation.
+
+  kvDirectBinding @53 :Bool
+      $compatEnableFlag("kvDirectBinding")
+      $experimental;
+  # Enables bypassing FL by translating pipeline tunnel configuration to subpipeline.
```

Comment:
Is this flag only used by the internal repo? If so, that should be indicated here.

Side note: it would be nice if we didn't have to list internal-only compat flags in workerd.

Reply:
Yes. Is the right way to specify that with a tag, prepending "internal", or noting that in the comment?

Reply:
Just in a comment, something like `"This flag is used only by the internal repo and not directly by workerd" or something similar